release_notes/42-release-notes.adoc

```diff
@@ -440,6 +441,17 @@ It contains the following bug fixes and changes:
 
 * A new default policy has been added, "Rapid Reset: Denial of Service Vulnerability in HTTP/2 Protocol". This policy alerts on deployments with images containing components that are susceptible to a Denial of Service (DoS) vulnerability for HTTP/2 servers, as described in link:https://access.redhat.com/security/cve/cve-2023-44487[CVE-2023-44487] and link:https://access.redhat.com/security/cve/cve-2023-39325[CVE-2023-39325]. This policy applies to the build or deploy life cycle stage.
 
+
+[id="resolved-in-version-4203_{context}"]
+=== Resolved in version 4.2.3
+
+*Release date:* 15 November 2023
+
+This release of {product-title-short} includes updates to {op-system-base-full} base images and includes the following fixes:
+
+* All containers have been rebuilt and now include container CVE fixes for link:https://access.redhat.com/security/cve/cve-2023-44487[CVE-2023-44487]: Flaw in handling multiplexed streams in the HTTP/2 protocol and link:https://access.redhat.com/security/cve/CVE-2023-40217[CVE-2023-40217]: Python 3 ssl.SSLSocket vulnerability.
```

Comment:
The 4.2.2 containers are currently not marked as being affected by CVE-2023-44487.